app/helpers/collections_cmr_helper.rb

```diff
@@ -0,0 +1,23 @@
+module CollectionsCmrHelper
```

Comment:
Per Charles comment about helpers largely being for views, we should discuss if/where we want to make this function available without having to manage two copies of it.

Reply:
this is very subjective/semantic, but I think `CmrCollectionsHelper` might be a little better (no need to change if you disagree)

Reply:
I think this can be moved into the `concerns` folder as it's not being used for views